subprojects/core/src/integTest/groovy/org/gradle/api/tasks/CachedTaskExecutionIntegrationTest.groovy

```diff
@@ -438,7 +436,7 @@ class CachedTaskExecutionIntegrationTest extends AbstractIntegrationSpec impleme
         withBuildCache().succeeds("cacheable")
 
         when:
-        gradleUserHome.deleteDir() // nuke the cache
+        gradleUserHome.deleteDir() // nuke the file snapshot cache
```

Comment:
Seems to work: https://builds.gradle.org/viewLog.html?buildTypeId=Gradle_Branches_CoveragePhase_WindowsCoverage_WindowsJava17DaemonIntegrationTe_7&buildId=2815424